app/models/vote.rb

```diff
@@ -125,6 +125,11 @@ def self.maximum_vote
     MAXIMUM_VOTE
   end
 
+  # minimum owner's vote needed to display owner_id
+  def self.owner_id_min_confidence
+    min_pos_vote
```

Comment:
This seems a bit odd to me given that min_pos_vote is just returning a constant.  I've been tend towards eliminating constants like that and just returning the value from the function.  At this point I would probably replace all the ALL_CAPS constants with function that simply return the number.  Then your code could stand as it is since conceptually it's a distinct value from min_pos_vote.  However, none of this is blocking accepting the pull as it is.